app/src/main/res/layout/activity_main.xml

```diff
@@ -1,12 +1,79 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.coordinatorlayout.widget.CoordinatorLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:card_view="http://schemas.android.com/apk/res-auto"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:id="@+id/coordinator_layout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    tools:ignore="SpUsage, ContentDescription, RtlHardcoded"
     >
 
-    <include layout="@layout/lists_with_add_content"/>
+    <LinearLayout
+        android:id="@+id/focal_point"
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:focusable="true"
+        android:focusableInTouchMode="true"
+        >
+
+        <FrameLayout
+            android:layout_width="match_parent"
+            android:layout_height="50dp"
+            android:layout_marginTop="8dp"
+            android:layout_marginLeft="8dp"
+            android:layout_marginRight="8dp"
+            >
+
+            <androidx.cardview.widget.CardView
+                android:id="@+id/add_item"
+                android:layout_gravity="end|center_vertical"
+                android:layout_width="50dp"
+                android:layout_height="match_parent"
+                card_view:contentPadding="5dp"
+                card_view:cardUseCompatPadding="true"
+                card_view:cardBackgroundColor="@color/app_blue"
+                card_view:cardCornerRadius="3dp"
+                card_view:cardElevation="2dp"
+                >
+
+                <ImageView
+                    android:id="@+id/plus_icon"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    />
+            </androidx.cardview.widget.CardView>
+
+            <EditText
+                android:id="@+id/item_name_input"
+                android:padding="8dp"
+                android:layout_gravity="center_vertical"
+                android:textColor="@color/dark_gray"
+                android:textSize="18dp"
+                android:background="@drawable/add_item_border"
+                android:layout_marginRight="55dp"
+                android:singleLine="true"
+                android:imeOptions="actionDone"
+                android:layout_width="match_parent"
+                android:layout_height="44dp"
+                android:hint="@string/add_list_hint"
+                />
+        </FrameLayout>
+
+        <TextView
+            android:id="@+id/no_content"
+            style="@style/empty_state_text"
+            />
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/names_recycle_view"
```

Comment:
nit: I'm not a fan of naming things after the type of view; it doesn't convey new information since we will bind this to a RecyclerView. I think a name like "user_lists" is better; it says what this view contains (the lists of the user).